package/MDAnalysis/coordinates/GRO.py

```diff
@@ -237,26 +239,30 @@ class GROWriter(base.WriterBase):
     }
     fmt['xyz_v'] = fmt['xyz'][:-1] + "{vel[0]:8.4f}{vel[1]:8.4f}{vel[2]:8.4f}\n"
 
-    def __init__(self, filename, convert_units=None, **kwargs):
+    def __init__(self, filename, convert_units=None,  **kwargs):
         """Set up a GROWriter with a precision of 3 decimal places.
 
         :Arguments:
-           *filename*
-              output filename
+            *filename*
+                output filename
+
+            *n_atoms*
+                int (optional)
+
         """
         self.filename = util.filename(filename, ext='gro')
+        self.n_atoms = kwargs.get('n_atoms', None)
```

Comment:
@kain88-de I think we can also get rid of `**kwargs`. It is not being used anywhere in the code now.